### DISCUSSION TITLE
Renv & deploying to shiny-server

### DISCUSSION THREAD

**Discussion by ElianHugh** [Appsilon/rhino#399] [Q&A]
I was wondering if there was a recommended way of deploying rhino apps via  open-source shiny server, particularly in relation to renv? I've found that in order to deploy properly, I have to `renv::restore()` and then immediately `renv::isolate()`. Having to call isolate every time seems like an extra step and I haven't seen anything about it mentioned in the rhino docs -- perhaps I'm going about this the wrong way?

**Comment by GuopingYe** [comment]
echo this issue, i'm struggling with Dockerfile now
